assets/thanos-querier/deployment.yaml

```diff
@@ -182,6 +188,9 @@ spec:
         - mountPath: /etc/kube-rbac-proxy
           name: secret-thanos-querier-kube-rbac-proxy-rules
       priorityClassName: system-cluster-critical
+      securityContext:
```

Comment:
this is breaking the deployment of Thanos querier. From https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_cluster-monitoring-operator/1264/pull-ci-openshift-cluster-monitoring-operator-release-4.8-e2e-agnostic-operator/1418233367471591424/artifacts/e2e-agnostic-operator/gather-extra/artifacts/deployments.json

```
"pods \"thanos-querier-6fdf64dc78-\" is forbidden: unable to validate against any security context constraint: [provider \"anyuid\": Forbidden: not usable by user or serviceaccount, provider restricted: .spec.securityContext.fsGroup: Invalid value: []int64{65534}: 65534 is not an allowed group, spec.containers[0].securityContext.runAsUser: Invalid value: 65534: must be in the ranges: [1000440000, 1000449999], spec.containers[1].securityContext.runAsUser: Invalid value: 65534: must be in the ranges: [1000440000, 1000449999], spec.containers[2].securityContext.runAsUser: Invalid value: 65534: must be in the ranges: [1000440000, 1000449999], spec.containers[3].securityContext.runAsUser: Invalid value: 65534: must be in the ranges: [1000440000, 1000449999], spec.containers[4].securityContext.runAsUser: Invalid value: 65534: must be in the ranges: [1000440000, 1000449999], provider \"nonroot\": Forbidden: not usable by user or serviceaccount, provider \"hostmount-anyuid\": Forbidden: not usable by user or serviceaccount, provider \"machine-api-termination-handler\": Forbidden: not usable by user or serviceaccount, provider \"hostnetwork\": Forbidden: not usable by user or serviceaccount, provider \"hostaccess\": Forbidden: not usable by user or serviceaccount, provider \"node-exporter\": Forbidden: not usable by user or serviceaccount, provider \"privileged\": Forbidden: not usable by user or serviceaccount]"
```